packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3523,18 +3523,19 @@ export default class Meeting extends StatelessWebexPlugin {
     };
 
     return Media.getDisplayMedia(shareConstraints, this.config)
-      .then((shareStream) => {
-        this.updateShare({
-          sendShare: true,
-          receiveShare: this.mediaProperties.mediaDirection.receiveShare,
-          stream: shareStream
-        });
-      })
+      .then((shareStream) => this.updateShare({
+        sendShare: true,
+        receiveShare: this.mediaProperties.mediaDirection.receiveShare,
+        stream: shareStream
+      }))
       .catch((error) => {
         // Whenever there is a failure when trying to access a user's display
         // report it as an operational metric
         // This gives visibility into common errors and can help
         // with further troubleshooting
+
```

Comment:
This will now catch an error on both `getDisplayMedia` and `updateShare`, does the code in the catch still apply or does it need to be updated?

Reply:
For now yes we need metrics for the share failures so we are going to use this event and rename it something else laters rather then getDisplayMedia as its a single feature.